avm-transpiler/src/opcodes.rs

```diff
@@ -72,14 +72,11 @@ pub enum AvmOpcode {
     // Misc
     DEBUGLOG,
     // Gadgets
-    KECCAK,
     POSEIDON2,
```

Comment:
If you have to do any other change, consider renaming POSEIDON2 to POSEIDON2PERM or similar, it's something I have on my wishlist